sol/private/sol_binary.bzl

```diff
@@ -131,6 +135,13 @@ def _run_solc(ctx):
         progress_message = "solc compile " + outputs[0].short_path,
     )
 
+    # Write a remappings.txt compatible with Forge
+    outputs.append(ctx.actions.declare_file("remappings.txt"))
```

Comment:
should it be a pre-declared output so you can reference it with a label?

Reply:
I've allowed the user to specify it as an optional attribute instead of hard-coding in an `outputs` dict for the `rule` declaration. Some may wish to have it live side-by-side with a vanilla Forge setup, in which case a `write_source_files({"remappings.txt": "foo"})` will be necessary, but I'd also like to create `sol_forge` rules in the future and those may just wish to specify the output directly as `remappings.txt`.